greenheart/simulation/greenheart_simulation.py

```diff
@@ -346,7 +346,7 @@ def setup_greenheart_simulation(config: GreenHeartSimulationConfig):
                 )
             ):
                 config.hopp_config["technologies"]["wind"]["fin_model"]["system_costs"][
-                    "om_fixed"
```

Comment:
Should `om_fixed` be used anywhere? Or is it just an incorrect value/not needed for our analyses?